### ISSUE TITLE
Version history doesn't work in VS2017

### ISSUE THREAD

**Issue by lvtgef3** [AmpScm/AnkhSVN#8]
After the automatic update today (18.05.2021) the version history doesn't work any more in VS2017 professional under Windows 10.

**Comment by rhuijben** [comment]
Can you explain what you see that doesn't work? If you see an error, please post that here.

And also please verify that you have AnkhSVN 2.9.51 (or later), see Extensions or the About dialog. We issues multiple releases in the last 24 hours.

**Comment by lvtgef3** [comment]
No error appears. The history window opens but there are no items. I use AnkhSVN 2.9.5.1

**Comment by fxandrei** [comment]
We have the same problem on multiple clients.  The window appears, no error, just nothing shows up.
Are there any logs anywhere that we can check ?

**Comment by Bahamuttg** [comment]
Same...

**Comment by rhuijben** [comment]
Could one of you having this issue check if the VSIX on https://github.com/AmpScm/AnkhSVN/actions/runs/853680940 fixes your issue?

It looks like this fixes this for me... Surprised that the same code works on VS2019.

**Comment by jadonclegg** [comment]
> Could one of you having this issue check if the VSIX on https://github.com/AmpScm/AnkhSVN/actions/runs/853680940 fixes your issue?
> 
> It looks like this fixes this for me... Surprised that the same code works on VS2019.

The 2017-2019 version fixes the subversion history for me in VS 2017

**Comment by jadonclegg** [comment]
While the version history is fixed, if I try to go to Pending Changes -> Update -> Merge Solution, and keep the default 'Merge a range of revisions' option selected, and hit next, VS freezes and eventually crashes with no error window.

**Comment by rhuijben** [comment]
@jadonclegg I separated this from the original issue.
Later VS2017 and now VS2019 forced us to switch to a more aynchronous coding model, which somehow breaks some commands that we already implemented asynchronously... mysteriously for VS2017 only, while I would have expected VS2019 to be stricter.

I will see what I can do. I'm glad the most important commands are mostly working again with the new unified VS2017-2019 package. I hope we can switch back to slightly larger and better tested releases soon. Help would be very welcome as reviving the project is mostly a one man show now.
